doc/COMPILE.md

```diff
@@ -324,7 +324,8 @@ The following base flags are available:
 
 The user flags are (the order matters!):
 
-- `p` use passwd/ldap to lookup home directory using `getpwuid_r`
+- `p` use passwd/ldap to lookup home directory using `getpwnam_r` with the username from the POSIX environment variable LOGNAME
```

Comment:
Please leave the behavior of this flag unchanged. Using LOGNAME is less reliable.